addons/mail/tests/test_mail_render.py

```diff
@@ -27,49 +27,21 @@ def setUpClass(cls):
             'lang': 'fr_FR',
         })
 
-        # some jinja templates
-        cls.base_jinja_bits = [
```

Comment:
Instead of removing, you should probably define inline_template bits to use on non html fields (subject, ...) to test that it effectively works as intended.